scripts/sql-ing-lookup.py

```diff
@@ -0,0 +1,80 @@
+#%%
+# Define function to query details for ingredients
+def scmd_ingredient_query(ingredient_list, str_to_lower = True, wildcards = "both"):
+    '''
+    Define SQL query searching for all ingredients specified in 'ingredient_list'.
+    The returned data includes the following columns:
+    
+    Arguments:
+    ingredient_list {list of strings}: for example ["drug1", "drug2", "drug3"]
+    wildcards {string}: "prefix", "suffix", "both", "none"
+    '''
+    # Define initial SQL query needed to query and join data
+    sql_query_start_l1 = "SELECT 'vmp' AS type, CAST(vmp.id AS STRING) AS id, bnf_code, vmp.nm, ing.nm AS ingredient, ddd.ddd FROM dmd.vmp\n"
+    sql_query_start_l2 = "INNER JOIN dmd.vpi AS vpi ON vmp.id = vpi.vmp\n"
+    sql_query_start_l3 = "INNER JOIN dmd.ing as ing ON ing.id = vpi.ing\n"
+    sql_query_start_l4 = "LEFT JOIN dmd.ddd on vmp.id = ddd.vpid\n"
+
+    sql_query_start = sql_query_start_l1 + sql_query_start_l2 + sql_query_start_l3 + sql_query_start_l4
```

Comment:
why 4 strings joined rather than 1 string?

Reply:
Agreed. The names of these variables don't help me understand why they are different. If they are, then also consider using an f-string:

```python
first_name = "Dizzee"
last_name = "Rascal"
full_name = f"{first_name} {last_name}"  # "Dizzee Rascal"
```

Reply:
Given none of that is parameterised I'd be inclined to write it as a triple-quoted multiline string

```
sql_query_start = """
SELECT 
  'vmp' AS type,
  CAST(vmp.id AS STRING) AS id,
  bnf_code,
  vmp.nm,
  ing.nm AS ingredient,
  ddd.ddd 
FROM dmd.vmp"
```